### ISSUE TITLE
Installation on Intel Galileo

### ISSUE THREAD

**Issue by microspace** [monteslu/node-red-contrib-gpio#3]
Hello
I'm trying to install this module on Intel Galileo.
I've installed johnny-five, galileo-io, node-red-contrib-gpio with npm.
New nodes doesn't show in node-red editor. What am I doind wrong?

**Comment by monteslu** [comment]
johnny-five isn't required.

Can you confirm that you ran `npm i galileo-io node-red-contrib-gpio` from the node-red installation direcgtory?

**Comment by microspace** [comment]
yes, I confirm. But after installation .node-red directory is empty.

**Comment by microspace** [comment]
I've copied ~/node-module folder to ~/.node-red. Now I see nodes in editor, but now I get "error loading io class".

**Comment by microspace** [comment]
I launched npm from inside .node-red directory. Now I have a folder node-modules. Inside that folder there is three subfolders: mraa, galileo-io, node-red-contrib-gpio.
Here is what I get when I run nod-red:

```
21 May 18:16:43 - [info] User Directory : /home/root/.node-red
21 May 18:16:43 - [info] Flows file     : /home/root/.node-red/flows_galileo.json
21 May 18:16:43 - [info] Server now running at http://127.0.0.1:1880/
21 May 18:16:43 - [info] Starting flows
21 May 18:16:44 - [info] [nodebot:73e0337.f8c1fcc] error loading io class
21 May 18:16:44 - [info] Started flows
21 May 18:16:44 - [warn] [gpio in:sfg] TypeError: bad map
```

When I deploy flow, I get:

```
3.05.2015, 0:24:02[ac1d9427.53e268]
[msg] : error
TypeError: bad set
```

```
uname -a 
Linux galileo 3.8.7-yocto-standard #1 Fri Feb 6 08:29:08 GMT 2015 i586 GNU/Linux
21 May 18:23:51 - [info] Node-RED version: v0.10.6
21 May 18:23:51 - [info] Node.js  version: v0.10.35
```

Please, can you tell me a way to debug this problem?

**Comment by Vebryn** [comment]
Same problem, did you solve your issue ?

**Comment by microspace** [comment]
No, I didn't try since then. But now, I've got Intel Edison, and there is no such problem.

**Comment by Vebryn** [comment]
Ok, thank you for your answer ! I encount a similar issue. Node-RED wasn't able to load raspi-io. My build was malformed. I built it again. Now it's working.
